CONTRIBUTING.md

```diff
@@ -1,6 +1,7 @@
 # Contribution Guidelines
 This document is a living summary of conventions and best practices for development within Zowe CLI or development of Zowe CLI plug-ins.
 
+  - [SIGN ALL OF YOUR GIT COMMITS](#sign-all-of-your-git-commits)
```

Comment:
Thanks for addressing my earlier comments! Noticed that the link here doesn't work, because the header below doesn't include the word "git". Please update so that the header is consistent in all 3 places: the ToC name, the ToC link, and the actual header.